eq_translations/survey_schema.py

```diff
@@ -69,6 +70,13 @@ def get_core_pointers(self):
             pointers.extend(key_pointers)
         return pointers
 
+    def get_schema_description_pointer(self):
+        pointers = []
```

Comment:
```Python
return [key_pointer[0] for key_pointer in find_pointers_to(self.schema('description')]
```

Reply:
I've done it a bit different, shorter version. Thanks!